src/api.ts

```diff
@@ -13,3 +36,90 @@ export async function getPeople(): Promise<Person[]> {
     .then(() => fetch(API_URL))
     .then(response => response.json());
 }
+
+function getParent(people: Person[], parentName: string) {
+  return people.find(({ name }) => name === parentName);
+}
+
+export function addParent(people: Person[]) {
+  return people.map(person => {
+    let mother;
+    let father;
+
+    if (person.motherName) {
+      mother = getParent(people, person.motherName);
+    }
+
+    if (person.fatherName) {
+      father = getParent(people, person.fatherName);
+    }
+
+    return {
+      ...person,
+      mother,
+      father,
+    };
+  });
+}
+
+export const sortPeople = (people: Person[], sortParam: string) => {
+  if (sortParam) {
+    return [...people].sort((a, b) => {
+      switch (sortParam) {
+        case ('name'): {
+          return a[sortParam].localeCompare(b[sortParam]);
+        }
+
+        case ('sex'): {
+          return a[sortParam].localeCompare(b[sortParam]);
+        }
+
+        case ('born'): {
+          return a[sortParam] - (b[sortParam]);
+        }
+
+        case ('died'): {
+          return a[sortParam] - (b[sortParam]);
+        }
```

Comment:
```suggestion
        case (SortParam.born):
        case (SortParam.died): {
          return a[sortParam] - (b[sortParam]);
        }
```
You can combine cases if they do a similar job